mlflow/gateway/schemas/embeddings.py

```diff
@@ -11,7 +11,8 @@
 
 
 class RequestPayload(RequestModel):
-    input: Union[str, List[str]]
+    input: Union[str, List[int], List[str], List[List[int]]]
+    encoding_format: Optional[Literal["float", "base64"]] = "float"
```

Comment:
@danilopeixoto Do we need `encoding_format`? You can include it in the request?

Reply:
Is this vendor specific or part of OpenAI spec?
We added to support `embedding: Union[List[float], str]` typing.

Although, it is not required due to "allow extra parameters".

Reply:
To avoid sending it, we can set it to `encoding_format: Optional[Literal["float", "base64"]] = None` instead `= "float"`.

Reply:
We can support `embedding: Union[List[float], str]` without `encoding_format`, right?